modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/Constants.tsx

```diff
@@ -32,4 +32,15 @@ const PAGINATION_PROPS = {
 	},
 };
 
-export {API_URL, OBJECT_RELATIONSHIP, PAGINATION_PROPS};
+const SORTING = {
+	ASCENDING: {
+		label: Liferay.Language.get('ascending'),
+		value: 'asc',
+	},
+	DESCENDING: {
+		label: Liferay.Language.get('descending'),
+		value: 'desc',
+	},
+};
```

Comment:
I believe this can be in `Sorting.tsx`. In `Constants` are constants used in multiple components.